### DISCUSSION TITLE
Seeking Help with Key Remapping for Transitioning Keyboard Layouts

### DISCUSSION THREAD

**Discussion by guehnwyvarr** [LazyVim/LazyVim#2770] [Q&A]
I'm considering transitioning from the QWERTY layout to either DVORAK or COLEMAK/COLEMAK-DH due to experiencing RSI in my right hand. As a relatively new user of Lazyvim, I'm still learning its configurations and Vim itself. Specifically, I'm grappling with how to adapt Lazyvim's keymaps, especially the default 'hjkl' motions, to accommodate the new layout.

I've been using Lazyvim's custom keymaps designed for the QWERTY layout, and I'm uncertain about the best approach for switching layouts. Should I remap all keys to fit the new layout, or is it possible to retain Lazyvim's default keymaps for most functions while adjusting only those related to motions, buffer movements, and window movements?

I haven't yet explored creating custom keymaps or modifying existing ones. Any guidance or advice on how to approach this transition would be greatly appreciated. Thank you for your assistance.

**Comment by jannikbuschke** [comment]
I am wondering the same. As mentioned here https://www.reddit.com/r/neovim/comments/1216uc5/comment/jdknxpa/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button one approach is to not change mappings. Im experimenting with that approach now.
